index.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+entries:
+  boldreports:
+  - apiVersion: v2
+    appVersion: 5.3.10
+    created: "2023-12-16T17:26:20.3843848+05:30"
+    description: Make bolder business decisions with complete reporting solutions
+      and unleash the power of your data with paginated, printable reports and make
+      intelligent decisions.
+    digest: F06176FA9C35A366E962DF81993E33B6C101F27FC0E9860A010849824906EE07
+    home: https://www.boldreports.com/
+    icon: https://on-premise-demo.boldreports.com/reporting/images/application/boldreports/main_logo.svg
+    name: boldreports
+    type: application
+    urls:
+    - https://boldreports.github.io/bold-reports-kubernetes/boldreports-5.3.10.tgz
```

Comment:
Give KC id for customer related helm chart files.

Reply:
changed